examples/105-dotnet-SemanticKernel-TextCompletion/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.KernelMemory;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI.TextEmbedding;
+
+var endpoint = Env.Var("AOAI_ENDPOINT");
+var apiKey = Env.Var("AOAI_API_KEY");
+var chatDeployment = Env.Var("AOAI_DEPLOYMENT_CHAT");
+var embeddingDeployment = Env.Var("AOAI_DEPLOYMENT_EMBEDDING");
+
+var memory = new KernelMemoryBuilder()
+    .WithSemanticKernelTextGenerationService(new AzureOpenAIChatCompletionService(chatDeployment, "", endpoint, apiKey))
+    .WithCustomEmbeddingGeneration(new AzureOpenAITextEmbeddingGeneration(embeddingDeployment, "", endpoint, apiKey))
+    .Build<MemoryServerless>();
+
+// using document form 203-dotnet-using-core-nuget
+
+await memory.ImportDocumentAsync("sample-SK-Readme.pdf", documentId: "doc001");
```

Comment:
I'm getting `Microsoft.KernelMemory.KernelMemoryException: File not found: 'sample-SK-Readme.pdf'` here (this project doesn't see artifacts from example 203, why not include the PDF?)